### PR TITLE
Ability to choose custom signals for stop or kill process run by executo...

### LINKED ISSUE #41: Ability to set signal with which try to end processes before terminating them
right now we're stopping them by default with SIGTERM. Though in order to end processes more gracefully, we might need to be able to set other signal to end processes.

### PR DESCRIPTION
**Pull request by fizyk**
...r - closes #41

### CODE REVIEW

**Comment by fizyk** [comment]
This pull request will be updated once #42 will make it into master ;)

**Comment by tomislater** [comment]
:+1:

File: mirakuru/base.py [hunk 1]
```diff
@@ -167,15 +181,19 @@ def stopped(self):
...
             self.start()
 
-    def kill(self, wait=True):
+    def kill(self, wait=True, signal_kill=None):
         """
         Kill the process if running.
 
         :param bool wait: set to `True` to wait for the process to end,
             or False, to simply proceed after sending signal.
+        :param int signal_kill: sugnal used to kill process run by  executor.
```
Comment (pwilczynskiclearcode) [inline mirakuru/base.py]: two typos:

```
        :param int signal_kill: sugnal used to kill process run by  executor.
```


```
        :param int signal_kill: signal used to kill process run by executor.
```

File: tests/executors/test_executor.py [hunk 2]
```diff
@@ -11,6 +13,42 @@ def test_running_process():
...
+    """Start process and shuts it down using signal SIGQUIT."""
+    executor = Executor('sleep 300', signal_kill=signal.SIGQUIT)
+    executor.start()
+    assert executor.running() is True
+    executor.kill()
+    assert executor.running() is False
+
+
+def test_kill_custom_signal_kill():
+    """Start process and shuts it down using signal SIGQUIT passed to stop."""
```
Comment (pwilczynskiclearcode) [inline tests/executors/test_executor.py]: ```passed to stop``` or to kill?

File: mirakuru/base.py [hunk 3]
```diff
@@ -167,15 +181,19 @@ def stopped(self):
             yield
             self.start()
 
-    def kill(self, wait=True):
+    def kill(self, wait=True, signal_kill=None):
```
Comment (pwilczynskiclearcode) [inline mirakuru/base.py]: perhaps instead of `signall_kill`, `sig` would be a name verbose enough?

**Pull request merged**